### DISCUSSION TITLE
RISOLTO:integrazione mobile app errore installazione

### DISCUSSION THREAD

**Discussion by totorethebassist** [indomus/forum#2132] [HUB: Home Assistant]
buongiorno a tutti, ho effettuato la ricerca nei topic per vedere se l'argomento fosse stato gia trattato 
se c'è qualcosa e non l'ho visto mi scuso e vi chiedo di postarmi il link.

venendo al problema: 
ho installato home assistant core su raspberry pi3b+ con raspbian in ambiente venv

hass funziona perfettamente 

l'accesso da remoto va benissimo 
(notizia poco nota: per i clienti vodafone,  dovete sapere che la vodafone station ha un ddns interno che viene usato dall'app "Station" quando si collega da remoto visto che l'ip del modem non è statico, per trovarlo basta installare una app che fa i log dei domini sul traffico del cellulare, tipo nextDns, avviarla e poi, non connessi al wifi, aprire l'app station  e cercare la riga che finisce per .mynet.vodafone.it/.    a che pro? serve a risparmiare il vil danaro di un abbonamento ad un servizio di dns dinamico )

ho associato il mio account tuya iot con il mio progetto cloud e va benissimo

quando provo ad installare l'integrazione per l'app mobile mi restituisce questo errore : Impossibile caricare il flusso di configurazione: {"message":"Invalid handler specified"}

**Comment by totorethebassist** [comment]
inizio a muovermi meglio nell'interfaccia per cui ho trovato anche il log 
Logger: homeassistant.config_entries
Source: config_entries.py:810
First occurred: 08:32:51 (1 occurrences)
Last logged: 08:32:51

Error occurred loading configuration flow for integration mobile_app: IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE! Importing the numpy C-extensions failed. This error can happen for many reasons, often due to issues with your setup or how NumPy was installed. We have compiled some common reasons and troubleshooting tips at: https://numpy.org/devdocs/user/troubleshooting-importerror.html Please note and check the following: * The Python version is: Python3.9 from "/srv/homeassistant/bin/python" * The NumPy version is: "1.23.2" and make sure that they are the versions you expect. Please carefully study the documentation linked above for further help. Original error was: libcblas.so.3: cannot open shared object file: No such file or directory

**Comment by totorethebassist** [comment]
Risolto:
Dopo qualche ricerca ho imparato che HA ha uno script per fare il check della configurazione
solite operazioni per cambiare utente ed attivare venv
dopodiché : hass --script check_config 
Il check ha evidenziato un errore su NumPy 
la documentazione di NumPy riporta che lo stesso, necessita della libreria libatlas-base-dev che non è  installata , ne indicata nelle varie guide (scopiazzate e tradotte) di HA core in ambiente venv .
Quindi dal terminale non in ambiente virtuale (utente pi, per intenderci) si installa la libreria.

**Reply by snakuzzo** [reply]
Ok, ma io ti consiglierei di lasciar perdere venv proprio per ritrovarti ogni tanto in queste situazioni. Inizia ad usare docker (magari con plugin compose).

**Reply by totorethebassist** [reply]
grazie! prenderò in considerazione il tuo consiglio

**Reply by totorethebassist** [reply]
visto che siamo in argomento e visto che ho deciso la raspberry deve essere dedicata esclusivamente a ha, tra la "core" su doker e "os" in termini di stabilità e dispendio di risorse quale mi consigli. 
Poi il trovarmi in "queste situazioni" (cit.) ritengo sia il sale di questo ambiente.
Grazie

**Reply by indomus** [reply]
sono la stessa cosa. su raspi sceglierei core su docker solo per questioni di performance.
